marketplace/src/main/assemble/assembly.xml

```diff
@@ -1,31 +1,190 @@
 <project name="shibboleth-assembly" default="build" xmlns:nx="urn:nuxeo-build" xmlns:artifact="urn:nuxeo-artifact">
-  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build" />
-  <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact" />
+  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build"/>
```

Comment:
Missing space at the end before `/>` in the whole file, IDE configuration I think.